src/NewsService.Data/NewsRepository.cs

```diff
@@ -20,19 +22,19 @@ public NewsRepository(IDataProvider provider)
             _provider = provider;
         }
 
-        public void EditNews(DbNews news)
+        public bool EditNews(Guid newsId, JsonPatchDocument<DbNews> news)
         {
-            var dbNews = _provider.News
-                .AsNoTracking()
-                .FirstOrDefault(x => x.Id == news.Id);
+            var dbNews = _provider.News.FirstOrDefault(x => x.Id == newsId);
 
             if (dbNews == null)
             {
-                throw new Exception("News was not found.");
+                throw new NotFoundException("News was not found.");
```

Comment:
should we return "false" if dbNews == null?

Reply:
It will be the exception, there will not be any return if exception